app/src/main/java/net/tensory/rxjavatalk/data/DebtFeed.java

```diff
@@ -0,0 +1,67 @@
+package net.tensory.rxjavatalk.data;
+
+import android.util.Log;
+
+import net.tensory.rxjavatalk.models.DebtReport;
+import net.tensory.rxjavatalk.models.House;
+import net.tensory.rxjavatalk.models.Value;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+import io.reactivex.Observable;
+import io.reactivex.subjects.PublishSubject;
+
+public class DebtFeed {
+
+    private static final double MAX_DEBT_GOLD = 1000000;
+    private PublishSubject<Object> didUpdateValuesSubject = PublishSubject.create();
+    private PublishSubject<DebtReport> debtReportSubject = PublishSubject.create();
+
+    private Map<House, Value<Double>> debts = new HashMap<>();
```

Comment:
I"m thinking that we probably want 3 behavior subjects, so that we can observe just the updates for an individual house.  Then we shouldn't need the debtReportSubject or didUpdateValuesSubject.